spec/models/host_collection_spec.rb

```diff
@@ -22,44 +22,44 @@ module Katello
     describe "create should" do
       it "should create succesfully with an org (katello)" do
         grp = HostCollection.create!(:name => "TestHostCollection", :organization => @org)
-        grp.wont_be_nil
+        value(grp).wont_be_nil
       end
 
       it "should not allow creation of a 2nd host collection in the same org with the same name" do
         HostCollection.create!(:name => "TestHostCollection", :organization => @org)
         grp2 = HostCollection.create(:name => "TestHostCollection", :organization => @org)
-        grp2.new_record?.must_equal(true)
-        HostCollection.where(:name => "TestHostCollection").count.must_equal(1)
+        value(grp2.new_record?).must_equal(true)
+        value(HostCollection.where(:name => "TestHostCollection").count).must_equal(1)
       end
 
       it "should allow host collections with the same name to be creatd in different orgs" do
         @org2 = Organization.create!(:name => 'test_org2', :label => 'test_org2')
         HostCollection.create!(:name => "TestHostCollection", :organization => @org)
         grp2 = HostCollection.create(:name => "TestHostCollection", :organization => @org2)
-        grp2.new_record?.must_equal(false)
-        HostCollection.where(:name => "TestHostCollection").count.must_equal(2)
+        value(grp2.new_record?).must_equal(false)
+        value(HostCollection.where(:name => "TestHostCollection").count).must_equal(2)
       end
 
       it "should not allow unlimited_hosts=false and max_hosts to be nil at the same time" do
         create = lambda do
           HostCollection.create!(:name => "TestHostCollection", :organization => @org, :unlimited_hosts => false)
         end
-        create.must_raise(ActiveRecord::RecordInvalid)
+        value { create }.must_raise(ActiveRecord::RecordInvalid)
```

Comment:
I think this is the issue:
```suggestion
        value(create).must_raise(ActiveRecord::RecordInvalid)
```